web/scripts/users/loginComponent.js

```diff
@@ -35,6 +35,11 @@ module.exports = {
 
             storage.set('jwt', res.authToken)
             storage.set('user', res.user)
+
+            // Set username view
+            document.getElementById("username-view").innerHTML
```

Comment:
Please resolve the eslint errors. VS Code should be underlining in red what they are.

Reply:
👍  Looks like there was a plugin issue on my VS Code. It's been resolved.